registry/storage/registry.go

```diff
@@ -230,6 +230,7 @@ func (repo *repository) Manifests(ctx context.Context, options ...distribution.M
 		// 2.1.0 unintentionally linked into  _layers.
 		manifestRevisionLinkPath,
 		blobLinkPath,
+		artifactRefPath,
```

Comment:
Should this be conditionally added if the extension is enabled? Can we use options for this purpose?

Reply:
removed this code